person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = @id = Random.rand(1...1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    of_age? or @parent_permission
```

Comment:
- [ ] Hi @Nuri1977, Please let's stick to Ruby's best practices. The `and` and `or` keywords are banned. `and` and `or`, despite an apparent similarity to `&&` and `||`, have very different roles. `and` and `or` are control-flow modifiers like `if `and `unless`. Specifically, they have a much lower precedence. Always use `&&` and `||` instead.